demos/rocksdb/run_benchmark.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+
+# 1. Init Occlum Workspace
+rm -rf occlum_instance && occlum new occlum_instance
+cd occlum_instance
+
+# 2. Copy files into Occlum Workspace and build
+rm -rf image
+copy_bom -f ../rocksdb.yaml --root image --include-dir /opt/occlum/etc/template
+
+new_json="$(jq '.resource_limits.user_space_size = "3000MB" |
+                .resource_limits.kernel_space_heap_size ="1024MB" |
+                .resource_limits.kernel_space_stack_size ="4MB" |
+                .process.default_heap_size = "128MB" |
```

Comment:
Can we remove the ".process.default_heap_size = "128MB" and .process.default_mmap_size = "2000MB" in the config if possible?

Reply:
Done